src/Analyzers/CSharp/Analyzers/RemoveUnnecessaryNullableDirective/NullableImpactingSpanWalker.cs

```diff
@@ -97,6 +97,10 @@ private static bool IsLanguageRestrictedToNonNullForm(TypeSyntax node)
             return true;
         }
 
+        // An attribute name cannot itself be nullable.  `[CLSCompliant?]` is not a thing.
+        if (node.IsParentKind(SyntaxKind.Attribute))
```

Comment:
An attribute can have a constructor parameter/property initializer, which can lead to warnings after nullable directive removed, e.g. `[MyAttr(null, Prop = null)]` with `MyAttr` declared as
```cs
#nullable enable

class MyAttr : Attribute
{
    public object Prop { get; set; }

    public MyAttr(string s) { }
}
```
Does this change account for that? Add test cases for this